apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.py

```diff
@@ -130,32 +130,29 @@ def render(element_html: str, data: pl.QuestionData) -> str:
                         initial_content = empty_diagram
                 else:  # submission
                     initial_content = data["submitted_answers"][drawing_name]
-            else:  # not gradable
-                if fresh and source_available:
-                    initial_content = load_file_content(element, data)
-                else:
-                    raise unreachable
+            elif fresh and source_available:
+                initial_content = load_file_content(element, data)
+            else:
+                raise unreachable
 
         case "answer":
             if gradable:
                 show_widget = False
-            else:  # not gradable
-                if fresh and source_available:
-                    initial_content = load_file_content(element, data)
-                else:
-                    raise unreachable
+            elif fresh and source_available:
+                initial_content = load_file_content(element, data)
+            else:
+                raise unreachable
```

Comment:
should probably fixup this `raise unreachable` later.

Reply:
Should be able to replace this with an assert_never.